hpOneView/oneview_client.py

```diff
@@ -225,12 +225,20 @@ def from_environment_variables(cls):
         auth_login_domain = os.environ.get('ONEVIEWSDK_AUTH_LOGIN_DOMAIN', '')
         password = os.environ.get('ONEVIEWSDK_PASSWORD', '')
         proxy = os.environ.get('ONEVIEWSDK_PROXY', '')
-
-        config = dict(ip=ip,
-                      image_streamer_ip=image_streamer_ip,
-                      api_version=api_version,
-                      credentials=dict(userName=username, authLoginDomain=auth_login_domain, password=password),
-                      proxy=proxy)
+        sessionID = os.environ.get('ONEVIEWSDK_SESSIONID', '')
+
+        if sessionID:
```

Comment:
how about:
```python
             config = dict(ip=ip,
                          image_streamer_ip=image_streamer_ip,
                          api_version=api_version,
                          credentials=dict(userName=username, authLoginDomain=auth_login_domain, password=password),
                          proxy=proxy)

            if sessionID:
              config['credentials'] = dict(sessionID=sessionID)
```
and drop the `else`? Should be a bit more compact and achieve the same.

P.S. This if *might* not even be necessary, since [this line](https://github.com/HewlettPackard/python-hpOneView/pull/335/files#diff-3c96d359db33fb08732f9084cb31140cR443) also has a similar if.
Was this tested with all configs inside credentials at once?

As in:
```python
credentials=dict(userName=username, authLoginDomain=auth_login_domain, password=password, sessionID=sessionID),
```

Reply:
Yes, since body (credentials) is not used for the PUT endpoint, it works. Changed.